### DISCUSSION TITLE
4 speakers on one zone

### DISCUSSION THREAD

**Discussion by suttin** [micro-nova/AmpliPi#281] [General] [answered]
I saw the question about a single channel in a zone, would there be any problems using 4 speakers in a single zone? I'd really hate to have to buy a zone expander if I don't need to.

**Comment by gorski123** [comment, marked as answer]
HI Suttin,
It depends. If you have two sets of 8 ohm speakers and you connect them in parallel (Connect two speakers to the right channel in parallel and two speakers to the left channel in parallel) AND the speakers are all 8 ohm, it should not be a problem. The minimum impedance the amplifiers can handle is 4 ohms, as long as you stay at or above that you should be fine. You may be able to add more than 2 sets of speakers to a single zone, but you would need a device that corrected the impedance to present an impedance of 4 or more ohms to the amps. Here is one on Amazon:

https://www.amazon.com/Monoprice-108230-8-Channel-Speaker-Selector/dp/B005E2YIAC

If you do go this route, make sure you buy one of the versions of this that states it has impedance matching circuity built in to make sure the total impedance stays at or above 4 ohms.
~Jason

**Comment by suttin** [comment]
Thanks for the answer. 

I know we are now well outside of the scope of AmpliPi, but two of the speakers I'm installing have this listed for the Impedance: 8 Ω Nom 4.5 Ω Min. Would those also be safe took hook up in parallel? Electronics always confuse me. 

Thanks again!

**Comment by bs42** [comment]
In my main bedroom I hooked up 4 speakers, I ran them serial to increase the Impedance to 16 instead of reduce down to 4 with parallel.  The AmpliPi has had no trouble driving them, at less than 50% volume setting I get plenty of sound.
